scalability/Makefile

```diff
@@ -220,16 +222,23 @@ core-experiment:
 	fi ; \
 	for program in $$EXPERIMENT_SET ; \
 	do \
-		for ext in $$EXPERIMENT_TYPE_SET ; \
-		do \
-			make $$program$$ext ; \
-		done ; \
+		if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
+			for ext in $$EXPERIMENT_TYPE_SET ; \
+			do \
+				make $$program$$ext ; \
+			done ; \
+		fi
 	done
 
 # This target is used for collecting data from the experiment's result into experiment.csv file.
 ${EXPERIMENT_CSV}: core-experiment
 	( cd ../utils ; CC=${CC} make treecmp ) 
-	echo "Directory,Program Options,klee-stats Time (s),KLEE info Elapsed (s), #instructions,#error,#completed paths,#error paths,#subsumed paths,#program exit paths,Average branching depth of completed paths,Average branching depth of subsumption paths,Average instructions of completed paths, Average instructions of subsumed paths,Icov,Bcov,llvm-cov line coverage,LOC,Time for actual solver calls in subsumption check (ms),Number of solver calls for subsumption check,Number of solver calls for subsumption check that resulted in subsumption failure,Average table entries per subsumption checkpoint,Average solver calls per subsumption check,Number of subsumption checks,KLEE paths saved,Tracer-X and not KLEE paths,KLEE and not Tracer-X paths,KLEE paths before last Tracer-X path (-1 if unreached)" >> $@
+	if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
```

Comment:
@rasoolmaghareh Similar to the comment for coreutils/Makefile, I think it is preferable to write:
```
echo -n "Directory, ..., Bcov,"
if [ .. ] ; then \
   echo -n "llvm-cov line coverage," ; \
fi
echo "LOC, ..."
```
so that if we want to change a field's title, there is only one place to change it.